core/src/main/java/com/linecorp/armeria/client/endpoint/EmptyEndpointGroupException.java

```diff
@@ -32,12 +35,25 @@
      * Returns an {@link EmptyEndpointGroupException} which may be a singleton or a new instance, depending on
      * {@link Flags#verboseExceptionSampler()}'s decision.
      */
-    public static EmptyEndpointGroupException get() {
+    public static EmptyEndpointGroupException get(@Nullable EndpointGroup endpointGroup) {
         return Flags.verboseExceptionSampler().isSampled(EmptyEndpointGroupException.class) ?
```

Comment:
How about always creating `EmptyEndpointGroupException` if `endpointGroup` is not null?